olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -783,21 +815,46 @@ spec:
       type: MutatingAdmissionWebhook
       deploymentName: knative-openshift
       admissionReviewVersions:
+        - v1alpha1
         - v1beta1
       containerPort: 9876
       failurePolicy: Ignore
       rules:
         - apiGroups:
             - operator.knative.dev
           apiVersions:
-            - v1alpha1
+            - v1beta1
           operations:
             - CREATE
             - UPDATE
           resources:
             - knativeservings
       sideEffects: None
       webhookPath: /mutate-knativeservings
+    - generateName: conversion.knativeservings.operator.knative.dev
```

Comment:
This all was added manually,
and the "matching" conversion was removed from the actual CRD, and put here.

Mind adding some sort of comment for the why? Not sure that's really very transparent - at least I stumbled on this when reviewing the branch :smile_cat:

Reply:
Yeah this comes from following the guide [here](https://olm.operatorframework.io/docs/advanced-tasks/adding-admission-and-conversion-webhooks/), [ocp docs](https://docs.openshift.com/container-platform/4.10/operators/operator_sdk/osdk-generating-csvs.html#olm-defining-csv-webhook_osdk-generating-csvs) and the discussion [here](https://coreos.slack.com/archives/C3VS0LV41/p1659474954314299). We don't auto-generate this part, not using the operator-sdk or anything.